app/screens/Map.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { Font } from 'expo'
+import openMap from 'react-native-open-maps';
```

Comment:
This looks like it's not being used. I don't think we have `react-native-open-maps` in the repo. Can this be removed or can you help me understand it's purpose?

Reply:
Whoops, that shouldn't be there. That's part of another feature branch I'm working on. That's what happens when you commit after midnight. Removing.